Autopilot/TelemetryManager/telemetryManager.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "telemetryStateManager.hpp"
+
+class telemetryState;
+
+class telemetryManager
+{
+    public:
+        telemetryManager();
+        inline telemetryState* getCurrentState() const{return currentState;}
```

Comment:
Okay nvm looks like we have support for it now... would've saved me a lot of hassle if this was the case 3 months ago 🤔
CI is failing because of an include error on the unit tests.